practice/react-practice-one/src/components/Toasts/Toast.jsx

```diff
@@ -0,0 +1,28 @@
+/* Import dependencies */
+import PropTypes from 'prop-types';
+import { useEffect } from 'react';
+
+/* Import CSS */
+import './toast.css';
+
+/* Import constants */
+import { STATUS, MESSAGES } from '../../constants';
+
+const Toast = ({ message, name, duration = 3000, onClose }) => {
+  useEffect(() => {
+    const timer = setTimeout(onClose, duration);
+
+    return () => clearTimeout(timer);
+  }, [duration, onClose]);
+
+  return <div className={`toast ${name}`}>{message}</div>;
+};
+
+Toast.propTypes = {
+  message: PropTypes.oneOf(Object.values(MESSAGES)),
+  name: PropTypes.oneOf(Object.values(STATUS)),
```

Comment:
@JennyDoJD how about `type` instead of name? btw please update the STATUS to TYPE or something like that

Reply:
I fixed it